### DISCUSSION TITLE
Prisma Accelerate - 502 / 503

### DISCUSSION THREAD

**Discussion by cipriancaba** [prisma/prisma#20745] [Q&A] [answered]
### Bug description

Not working on 5.1.1

### How to reproduce

Follow the instructions

### Expected behavior

I can query the data

### Prisma information

Calling https://accelerate.prisma-data.net/5.1.1/**/graphql
Invalid `prisma.order.findMany()` invocation:
Unknown server error: {"type":"UnknownTextError","body":"<html><body><h1>503 Service Unavailable</h1>\nNo server is available to handle this request.\n</body></html>\n"}

### Environment & setup

macos
postgres
node 18

### Prisma Version

```
5.1.1
```

**Comment by ankur-arch** [comment]
The [conversation](https://prisma.slack.com/archives/C044YLSN49E/p1692360771139639) is being continued on our public Slack channel.

**Comment by ankur-arch** [comment, marked as answer]
The problem occurred because a private database connection was being used. Currently, Accelerate does not support static IPs.
